README.md

```diff
@@ -7,44 +7,43 @@ panelGC effectively quantifies and monitors GC bias in hybridization capture seq
 amd64, arm64v8, ppc64le, s390x ([more info](https://github.com/docker-library/official-images#architectures-other-than-amd64))
 
 ## Dependencies
-- Nextflow (tested with 21.10.6)
 - Singularity (tested with 2.5.1)
 
-Containerized dependencies (no installation required):
-bedtools 2.30.0
-r-base 4.3.2 argparser 0.7.1 BiocManager 1.30.22 GenomicRanges 1.54.1 rtracklayer 1.62.0 tidyverse 2.0.0
-
 ## Installation
-- Install Nextflow (if not already installed): \
-Refer to official [Nextflow installation guide](https://www.nextflow.io/docs/latest/install.html)
-- Install Singularity (if not included with Nextflow): \
+- Install Singularity: \
 Refer to official [Singularity installation guide](https://docs.sylabs.io/guides/latest/user-guide/quick_start.html)
-- Clone the panelGC repository:
+
+## Usage
+### Clone Repository:
 ```bash
 git clone https://github.com/easygsea/panelGC.git
+cd panelGC
 ```
-
-## Usage
-Run panelGC with the following command:
+### Pull Singularity Image:
 ```bash
-nextflow run panelGC.nf \
-  --bam_directory_path /path/to/bam_files/ \
-  --bed_file_path /path/to/bed_file.bed \
-  --fasta_file_path /path/to/fasta_file.fa \
-  --out_dir /path/to/output_directory/
+singularity pull docker://murathangoktas/panelgc:latest
 ```
-Replace the paths with your actual data directories and file paths. Adjust [parameters](https://github.com/easygsea/panelGC/tree/main?tab=readme-ov-file#parameters) as needed.
-
-Alternatively, configure parameters in a JSON or YML file (see official [Nextflow parameters guide](https://training.nextflow.io/basic_training/config/#parameters)) and excecute the following command:
+### Basic Usage:
 ```bash
-nextflow run panelGC.nf -params-file params.json
+singularity exec \
+  -B $(pwd):/workspace panelgc_latest.sif \
+  nextflow run /workspace/panelGC.nf \
+  --bam_directory_path /workspace/bam_files/ \
+  --bed_file_path /workspace/bed_file.bed \
+  --fasta_file_path /workspace/fasta_file.fa \
+  --out_dir /workspace/output_directory/
 ```
-
-**Note:** If you have limited space in your working directory or prefer to store the Singularity container in a different location, you can set the 'NXF_SINGULARITY_CACHEDIR' environment variable. This variable allows you to specify a custom path for storing Singularity images. To use this feature, export the 'NXF_SINGULARITY_CACHEDIR' variable with your desired path before running the panelGC command. For example:
```

Comment:
Instruction regarding temporary working directory would still be useful. Can cite official manual regarding `--workdir` (https://docs.sylabs.io/guides/4.2/user-guide/cli/singularity_run.html#singularity-run)

Reply:
This note was about 'NXF_SINGULARITY_CACHEDIR' which is used only if Nextflow runs processes with a Singularity container. We deviated from that so this note doesnt apply now.

Reply:
I cant see the relevance of `--workdir` here. I never used it. If you dont agree can you share a note you suppose to add here?

Reply:
Sorry I misread this paragraph. I see the lines removed within nextflow.config correspondingly.

A note regarding `singularity exec` options would be useful:

**Note:** `singularity exec` provides additional options to help optimize the deployment of panelGC for your specific use. For more details, please refer to the [official guide](https://docs.sylabs.io/guides/latest/user-guide/cli/singularity_exec.html).

Reply:
Also nextflow:

**Note:** `singularity exec` and `nextflow` provide additional options to help optimize the deployment of panelGC for your specific use. For more details, please refer to the official guides: [singularity exec](https://docs.sylabs.io/guides/latest/user-guide/cli/singularity_exec.html) and [nextflow CLI reference](https://www.nextflow.io/docs/latest/reference/cli.html#options).